pkg/web/templates/partials/secure_envelope/secure_envelope_list.html

```diff
@@ -1,6 +1,6 @@
 {{ if .DecryptedEnvelopes }}
```

Comment:
I'd like to note that currently `.IsDecrypted` returns false so the UI is currently displaying the `Secure Envelope`.